yugabytedb_managed/assets/dashboards/yugabytedb_managed_overview.json

```diff
@@ -0,0 +1,2017 @@
+{
+  "title": "YugabyteDB Managed Overview",
+  "description": "This dashboard allows users to monitor the health and performance of their YugabyteDB Managed clusters.",
+  "widgets": [
+    {
+      "id": 6225493483652460,
+      "definition": {
+        "type": "image",
+        "url": "https://raw.githubusercontent.com/DataDog/integrations-extras/master/yugabytedb_managed/assets/logos/yugabytedb_managed_logo.svg",
```

Comment:
The logo needs to be either png, gif, or jpg. Please update the image to one of the listed formats

Reply:
Done